values.yaml

```diff
@@ -1,33 +1,43 @@
-octopus:
-  statefulSet:
-    annotations: {}
-    labels: {}
-  pods:
-    annotations: {}
-    labels: {}
+statefulSetLabel:
+  environment: prod
+
+podLabels:
+  environment: prod
+
   # The Octopus server image. Visit https://hub.docker.com/r/octopusdeploy/octopusdeploy for the available versions.
   image: octopusdeploy/octopusdeploy:2020.3.1-linux
-  # Set this to 1 if you do not have a HA license.
-  replicaCount: 1
+
+  # Set this to 1 if you do not have a HA license. If you want to run Octopus Deploy with HA and have an HA license, change the replica count
+  replicaCount: 2
```

Comment:
HA by default imposes a lot of extra configuration that I suspect most people won't initially want. I would leave this value with a default of `1`.

Reply:
Yeah, that definitely makes sense. I've changed the value back to 1.